### PR TITLE
Fixed duplication of expected start and end date

### PR DESCRIPTION
**Pull request by vigilantee**
Marked the expected and end date of tasks.

### CODE REVIEW

**Comment by Alchez** [comment]
@vigilantee, I think your editor might have auto-saved the JSON, which is causing the files here to be re-formatted. Can you save the doctypes again to re-generate the JSON properly?

File: erpnext/projects/doctype/task/task.json [hunk 1]
```diff
@@ -201,7 +202,6 @@
   {
    "fieldname": "description",
    "fieldtype": "Text Editor",
-   "in_preview": 1,
```
Comment (Alchez) [inline erpnext/projects/doctype/task/task.json]: Keep this

File: erpnext/projects/doctype/task/task.json [hunk 2]
```diff
@@ -325,7 +325,6 @@
    "options": "Department"
   },
   {
-   "fetch_from": "project.company",
```
Comment (Alchez) [inline erpnext/projects/doctype/task/task.json]: Keep this

**Review by Alchez — CHANGES_REQUESTED** [review]

File: erpnext/projects/doctype/task/task.json (hunk 2 above)
Comment (vigilantee) [reply erpnext/projects/doctype/task/task.json]: Done.

File: erpnext/projects/doctype/task/task.json (hunk 1 above)
Comment (vigilantee) [reply erpnext/projects/doctype/task/task.json]: Done.

**Review by Alchez — APPROVED** [review]

**Pull request merged**